python/tests/scripts/a1_symbols.metta

```diff
@@ -18,6 +18,20 @@
   (leaf1
    (leaf0 leaf1)))
 
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+; `assertContains` Checks if the second argument is 
+; one of the expected results of the first   
+; expression's evaluation
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+!(assertContains
+  (match &self (($x leaf2) leaf3) $x)
+  leaf1)
```

Comment:
I don't think the tests should be here, they should probably be in their own place, but I don't know where.  @vsbogd, what do you think?

Reply:
Usually I write unit tests for the stdlib in Rust. The Rust MeTTa runner is called with MeTTa program written as a string and result is compared to the expected atom. The only tricky part there is using `expr!` macros to create an expected atom. The macros has the following syntax:
- string in quotes means symbol
- parenthesis add nested expression
- identifier (without quotes) creates variable
- curly brackets add grounded atom

For instance: `expr!("sym" var ("nested" "expression"))` corresponds to the MeTTa `(sym $var (nested expression))`.

For example see link below or any other test which calls `run_program`: https://github.com/trueagi-io/hyperon-experimental/blob/bde1b596f4e63a26faf377902cda00915908d27e/lib/src/metta/runner/stdlib/mod.rs#L675-L703

Reply:
I definitely agree that tests for different asserts should not be in `a1_symbols.metta`. If one really wants these tests to be in `scripts` folder, one should possibly create a separate file and systematically cover different asserts there, so this file would simultaneously serve as a sort of tutorial about asserts and provide a good coverage on them. A stand-alone test for a particular assert should be somewhere else, e.g. in Rust tests for stdlib as @vsbogd said.